src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -689,4 +694,14 @@ else if (scoreLoc.isCone()) {
       }
     }
   }
+
+  public void rePrepScoreSequence(Lights lights) {
+    if (goalPosition == LiftPosition.PRE_SCORE_HIGH_CONE
+        || goalPosition == LiftPosition.PRE_SCORE_MID_CONE
+        || goalPosition == LiftPosition.SCORE_MID_CUBE
+        || goalPosition == LiftPosition.SCORE_HIGH_CUBE
+        || goalPosition == LiftPosition.SCORE_LOW) {
```

Comment:
Long (probably needlessly so) comment incoming:

Pretty sure this logic has a flaw. Because we only have the two pre-score positions used here (mid/high cone), if we are running it based only on arm position, we must use score mid/high cube and score low, as Ava did. However, those are *also* the positions that it is in when I am holding RT. So if I go to prep score, start holding RT, and then Luka changes the position, it will still go to the new position (if it is scoring low or a cube). At that point, I am still holding RT. If Luka changes it to another position that doesn't have a pre-score, that's fine; it should go right to the scoring position. However, if he changes it to mid/high cone, it will go to a pre-score position. But I'm still holding RT, so I can't get it to the actual scoring position, and if I let go of RT to re-press it it will run `rt.onFalse` and drop the object.

Reply:
You're right. Maybe we check the started score flag as well?